src/PowerShellEditorServices/Services/PowerShellContext/Handlers/GetCommentHelpHandler.cs

```diff
@@ -80,21 +79,27 @@ public async Task<CommentHelpRequestResult> Handle(CommentHelpRequestParams requ
                     vscodeSnippetCorrection: true,
                     placement: helpLocation));
 
-            string helpText = analysisResults?.FirstOrDefault()?.Correction?.Edits[0].Text;
+            string helpText = analysisResults?[0]?.Correction?.Edits[0].Text;
 
             if (helpText == null)
             {
                 return result;
             }
 
-            result.Content = ScriptFile.GetLinesInternal(helpText).ToArray();
+            List<string> helpLines = ScriptFile.GetLinesInternal(helpText);
 
             if (helpLocation != null &&
                 !helpLocation.Equals("before", StringComparison.OrdinalIgnoreCase))
             {
                 // we need to trim the leading `{` and newline when helpLocation=="begin"
                 // we also need to trim the leading newline when helpLocation=="end"
-                result.Content = result.Content.Skip(1).ToArray();
+                helpLines = helpLines.GetRange(1, helpLines.Count - 1);
```

Comment:
`GetRange` will duplicate the entire list into a new value, leaving two lists on the heap to garbage collect.

It might be better to use `RemoveAt`:

```csharp
helpLines.RemoveAt(0);
helpLines.RemoveAt(helpLines.Count - 1);
```